docs/docs/resources/redis.md

```diff
@@ -6,9 +6,14 @@ weight: 50
 
 Aiven for Redis®\* is a fully managed in-memory NoSQL database that you can deploy in the cloud of your choice to store and access data quickly and efficiently.
 
-!!! note
-    Before going through this guide, make sure you have a [Kubernetes cluster](../installation/prerequisites.md) with the operator installed (see instructions for [helm](../installation/helm.md) or [kubectl](../installation/kubectl.md)),
-    and a [Kubernetes Secret with an Aiven authentication token](../authentication.md).
+!!! warning "End of life notice"
+
+    In March 2024, a new licensing model was announced for Redis® that impacts the Aiven for Caching offering (formerly Aiven for Redis®). Aiven for Caching is entering its end-of-life cycle to comply with Redis's copyright and license agreements. From **February 15th, 2025**, it will not be possible to start a new Aiven for Caching service, but existing services up until version 7.2 will still be available until end of life. From **March 31st, 2025**, Aiven for Caching will no longer be available and all existing services will be migrated to Aiven for Valkey™. You can [upgrade to Valkey for free yourself](https://aiven.io/docs/products/caching/howto/upgrade-aiven-for-caching-to-valkey) before then.
+
```

Comment:
Hey Stacey, we added a deprecation warning here: https://github.com/aiven/aiven-operator/pull/879

It shows up in docs like this:

![image](https://github.com/user-attachments/assets/1217490a-4540-4593-8a19-a204d6a7f99d)


I think we could remove this part.

Reply:
Hey, yeah, I saw that, but I'd like to include this on both Redis pages and at least have the key dates here. I also wanted to improve the link to the upgrade instructions. I can remove the first sentence though.